### DISCUSSION TITLE
visual Studio query

### DISCUSSION THREAD

**Discussion by dbeaver12** [microsoft/vscode-discussions#665] [Extension Development QnA] [answered]
With ref to the software for Manchester Met university , we need to know the following queries  -
•	When was the software last updated?
•	Is the software still in active development?
•	Are the software provider committed to patching the software in the event of a security incident or known security vulnerabilities?

Thanks ,

Jesu
jesu.mathew@mmu.ac.uk

**Comment by alefragnani** [comment, marked as answer]
Hi @dbeaver12 ,

This kind of question would be better suited on VS Code repo, and BTW, I noticed you already did that (https://github.com/microsoft/vscode/issues/185849), which is great.

This Discussions repo is focused for extension development for now. And if you need the same answer from the extensions you are willing to install and use, I suppose you will have to ask each extension author individually (individuals or companies), as there is no kind of arrangement between extension authors and the extension ecosystem.

Hope this helps
